hist/hist/src/TGraph2D.cxx

```diff
@@ -986,6 +986,18 @@ void TGraph2D::CreateInterpolator(Bool_t oldInterp)
    }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Return pointer to function with name.
+///
+/// Functions such as TGraph2D::Fit store the fitted function in the list of
+/// functions of this graph.
+
+TF2 *TGraph2D::GetFunction(const char *name) const
+{
+   if (!fFunctions) return nullptr;
+   return (TF2*)fFunctions->FindObject(name);
```

Comment:
```suggestion
   return dynamic_cast<TF2*>(FindObject(name));
```

Reply:
I would do the safe thing here and use a dynamic_cast to `TF2`.

Also, we don't need to duplicate the find logic, since there is already `FindObject`. What do you think?